core/base/src/constants/modules.ts

```diff
@@ -0,0 +1,39 @@
+/*TODO
+ * The term module is a hot mess in the wormhole ecosystem:
+ *   governance actions have a module parameter:
+ *     - "Core" - https://github.com/wormhole-foundation/wormhole/blob/9e61d151c61bedb18ab1d4ca6ffb1c6c91b108f0/ethereum/contracts/Governance.sol#L21
+ *     - "TokenBridge" - https://github.com/wormhole-foundation/wormhole/blob/9e61d151c61bedb18ab1d4ca6ffb1c6c91b108f0/ethereum/contracts/bridge/BridgeGovernance.sol#L24
+ *     - "NFTBridge" - https://github.com/wormhole-foundation/wormhole/blob/9e61d151c61bedb18ab1d4ca6ffb1c6c91b108f0/ethereum/contracts/nft/NFTBridgeGovernance.sol#L23
+ *     - "WormholeRelayer" - https://github.com/wormhole-foundation/wormhole/blob/9e61d151c61bedb18ab1d4ca6ffb1c6c91b108f0/ethereum/contracts/relayer/wormholeRelayer/WormholeRelayerGovernance.sol#L43
+ *   while the core contract is actually called "Wormhole" in most platforms
+ *     - though in solana it's called bridge - https://github.com/wormhole-foundation/wormhole/tree/main/solana/bridge
+ *     - and in algorand it seems to be called wormhole_core - https://github.com/wormhole-foundation/wormhole/blob/main/algorand/wormhole_core.py
+ *     - and in EVM it resides directly in the contracts directory
+ *   the naming of the token bridge and the nft bridge seem to be a lot more consistent, though:
+ *     - for EVM the TokenBridge and NFTbridge reside in the "bridge" and "nft" directory respectively: https://github.com/wormhole-foundation/wormhole/tree/main/ethereum/contracts)
+ *   the WormholeRelayer resides in relayer/wormholeRelayer (only built for EVM so far)
+ * 
+ *   Within the solana directory, only the token bridge and the nft bridge are considered modules
+ *     (i.e. are in the modules directory: https://github.com/wormhole-foundation/wormhole/tree/main/solana/modules).
+ *   While in the JS SDK, the core bridge functionality is in the "bridge" directory
+ *     (notice the clash with EVM where bridge refers to the token bridge...).
+ * 
+ * Additionally, "modules" seems like a bad choice in a TS/JS context...
+ * 
+ * With all of this in mind: What should we name modules here?
+ * My preferred choice would be ["CoreBridge", "TokenBridge", "NftBridge", "Relayer"]
+ *   but ["Core", "TokenBridge", "NFTBridge", "WormholeRelayer"] seems to be more consistent given
+ *   current naming "conventions" 
+ */
+
+export const modules = [
+  "CoreBridge",
+  "TokenBridge",
+  "NftBridge",
+  "Relayer",
+//TODO "Connect"
```

Comment:
I wouldn't list Connect as a module, but rather a consumer of all of the above

Reply:
Connect to me is the token bridge relayer which fits the mold of a module:
It has its own set of contracts, payloads, etc.

Reply:
beyond the length, it's a bit ugly to have one module name also be the prefix of another module name... :confused: 

It's certainly much more descriptive though. I guess my thought here is: It's the module that powers the connect widget. I agree that in light of the name of the sdk "connect" is a disgustingly bad term - but imho it's the SDK who is at fault here <insert principal Skinner meme here>.

Reply:
In a similar vein, "Relayer" is pretty unspecific, but imho still better than what seems to be its canonical name atm i.e. "WormholeRelayer" (because in an SDK by Wormhole whose Relayer will it be?) or the alternative of "AutomaticRelayer" (as opposed to what sort of relayer? a non-automatic one? as in no relayer whatsoever?)

I feel like were being backed into a corner her by all the other terrible naming choices that have been made (a bit of a theme I suppose).